doc/other_modules/jenny/language/expressions/functions/misc.md

```diff
@@ -1,6 +1,25 @@
 # Miscellaneous functions
 
 
+## if(condition, then, else)
+
+This function implements the ternary-if condition, it is equivalent to the `?:` operator in C/C++.
```

Comment:
Isn't is better to compare it to the ternary operator in Dart?

Reply:
Oh, right, forgot about Dart 🤣